kymatio/scattering1d/frontend/numpy_frontend.py

```diff
@@ -46,7 +46,7 @@ def scattering(self, x):
         # treat the arguments
         if self.vectorize:
             size_scattering = precompute_size_scattering(
-                self.J, self.Q, self.T, max_order=self.max_order, detail=True)
+                self.J, self.Q, self.T, max_order=self.max_order)
```

Comment:
Could we not just do `size_scattering = self.num_coefficents`?

Reply:
this code block will disappear with #837. i wouldn't worry about it